File: ghoul.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading;
using System.Threading.Tasks;

namespace ghoul
{
    class Program
    {
        static int h = 1000;
        static void Main(string[] args)
        {
            Console.WriteLine(h);
            Thread.Sleep(1000);
            while (h > -1)
            {
                h = h - 7;
                Console.WriteLine(h);
                Thread.Sleep(50);
            }
            Console.ReadKey();
        }
    }
}

File: ghoul.vb
Imports System
Imports System.Threading

Module Program
    Sub Main(args As String())
        Dim h As Integer = 1000
        Console.WriteLine(h)
        Thread.Sleep(2000)
        While h > -1
            h -= 7
            Console.WriteLine(h)
            Thread.Sleep(50)
        End While
        Console.ReadKey()
    End Sub
End Module
